.golangci.yml

```diff
@@ -125,7 +125,7 @@ linters:
     - testableexamples
     # - thelper
     - unconvert
-    - unparam
+    # - unparam
```

Comment:
Why are we disabling this linter? 🤔

Reply:
@lovromazgon it was complaining about [logger.go:33](https://github.com/ConduitIO/conduit-connector-sdk/blob/32a20f262a14bd16e486f9da550f7bcb42e5446a/logger.go#L33) returning always nil: https://github.com/ConduitIO/conduit-connector-sdk/actions/runs/9782459734/job/27008810920#step:5:33

Reply:
Rightfully so, seems like we can remove the `error` returned there 😉

Reply:
@lovromazgon right. Hadn't realized this was a private function. For some reason I thought refactoring had other implications 🙃 . I'll refactor.